### PR TITLE
Django modified structure, and recieves a played match replay

### LINKED ISSUE #12: Receive a played match replay
The overlord will post a base 64 encoded replay of the played match

### PR DESCRIPTION
**Pull request by bcattaneo**
Fixes #12

### CODE REVIEW

File: Overmind/settings.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    },
+]
+
+WSGI_APPLICATION = 'Overmind.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/2.1/ref/settings/#databases
+
+DATABASES = {
```
Comment (MMandirola) [inline Overmind/settings.py:63]: Postgresql configuration, "psicopg2"

File: Overmind/urls.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 """Overmind URL Configuration
```
Comment (MMandirola) [inline Overmind/urls.py]: Cleanup boilerplate comment

File: Replays/prueba5.SC2Replay [hunk 3]
```diff
@@ -0,0 +1 @@
+hola
```
Comment (MMandirola) [inline Replays/prueba5.SC2Replay]: Add Replays/* to .gitignore

File: api/admin.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```
Comment (MMandirola) [inline api/admin.py:1]: Add admin config to watch replay records on django admin

File: api/models.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+class Replays(models.Model):
+    title = models.CharField(max_length=200)
+    base64_file = models.TextField()
+    extra = models.TextField()
+    date = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        #return '%s %s' % (self.title, self.base64_file)
```
Comment (MMandirola) [inline api/models.py]: Cleanup

File: api/resources.py [hunk 6]
```diff
@@ -0,0 +1,30 @@
+import base64
```
Comment (MMandirola) [inline api/resources.py]: Move to views.py

File: api/tests.py [hunk 7]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
```
Comment (MMandirola) [inline api/tests.py:5]: Add some tests

File: api/resources.py (hunk 6 above)
Comment (MMandirola) [reply api/resources.py]: Add tastypie and django==2.1 to a requirements.txt file

**Review by MMandirola — CHANGES_REQUESTED** [review]

File: Replays/prueba5.SC2Replay (hunk 3 above)
Comment (MMandirola) [inline Replays/prueba5.SC2Replay]: Remove replay file

File: Overmind/settings.py [hunk 8]
```diff
@@ -0,0 +1,122 @@
+"""
```
Comment (MMandirola) [inline Overmind/settings.py]: Investigate Local settings pattern https://techstream.org/Bits/Local-Settings-in-django

**Review by MMandirola — CHANGES_REQUESTED** [review]

**Review by MMandirola — APPROVED** [review]

**Comment by bcattaneo** [comment]
Also fixes: #6 #1 #9

**Pull request merged**